src/main/java/org/jenkinsci/plugins/lucene/search/databackend/SolrSearchBackend.java

```diff
@@ -388,7 +392,7 @@ public void cleanDeletedJobs(Progress progress) throws Exception {
         }
     }
 
-    public Set<String> getFacetsOfField(String fieldName) throws SolrServerException {
+    public Set<String> getFacetsOfField(String fieldName) throws SolrServerException, IOException {
```

Comment:
Technically it's a binary compatibility change. But Lucene upgrade requires it in any case :)